frontend/app/src/store/session/index.ts

```diff
@@ -157,8 +156,6 @@ export const useSessionStore = defineStore('session', () => {
     try {
       await usersApi.logout(get(username));
       set(logged, false);
-      updateDbUpgradeStatus();
-      updateDataMigrationStatus();
```

Comment:
In general, we don't really need to manually clear stores on logout, all of them should reset to defaults, (with the exception of a couple of properties in the main store(